### PR TITLE
New logic of LP3

### PR DESCRIPTION
**Pull request by AndreyAndreev**

### CODE REVIEW

File: src/Lykke.Service.LP3.DomainServices/Lp3Service.cs [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 using Lykke.Service.LP3.Domain.Assets;
 using Lykke.Service.LP3.Domain.Exchanges;
 using Lykke.Service.LP3.Domain.Orders;
 using Lykke.Service.LP3.Domain.Services;
 using Lykke.Service.LP3.Domain.Settings;
-using Lykke.Service.LP3.Domain.TradingAlgorithm;
 
 namespace Lykke.Service.LP3.DomainServices
 {
     public class Lp3Service : ILp3Service, IStartable, IDisposable
```
Comment (aushakov) [inline src/Lykke.Service.LP3.DomainServices/Lp3Service.cs]: Could you move SynchronizeAsync to the wrapper like in NE project?

**Review by aushakov — COMMENTED** [review]
Add descriptions for public API.
The "OrderBook" term means a collection of orders but you use it for settings to calculate orders.
Please, don't mix public and private methods in a class. Put private methods below to public. [Ordering Rules](http://stylecop.soyuz5.com/Ordering%20Rules.html).

File: client/Lykke.Service.LP3.Client/Models/Orders/LimitOrderError.cs [hunk 2]
```diff
@@ -11,6 +11,8 @@ public enum LimitOrderError
         EmptyOrderBook = 3,
         
         ExceedsSideSumVolume = 4,
+        
+        OrderBookIsDisabled = 5,
 
         LowBalance = 401,
```
[the remark this answers is not in the document]
Comment (AndreyAndreev) [reply client/Lykke.Service.LP3.Client/Models/Orders/LimitOrderError.cs]: fixed

File: client/Lykke.Service.LP3.Client/Models/Orders/LimitOrderPostModel.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    {
+        public string AssetPairId { get; set; }
+        
+        public decimal Price { get; set; }
+        
+        public decimal Volume { get; set; }
+        
+        public decimal Number { get; set; }
+        
+        public TradeType TradeType { get; set; }
```
[the remark this answers is not in the document]
Comment (AndreyAndreev) [reply client/Lykke.Service.LP3.Client/Models/Orders/LimitOrderPostModel.cs]: fixed

File: src/Lykke.Service.LP3.DomainServices/Lp3Service.cs (hunk 1 above)
Comment (AndreyAndreev) [reply src/Lykke.Service.LP3.DomainServices/Lp3Service.cs]: I think it's unnecessary in the current state of LP3 project to extract synchronisation logic into separate wrapper class as there is only one class with such sync logic and this is just an inner implementation of it.

**Review by aushakov — APPROVED** [review]

**Pull request merged**